x-pack/platform/plugins/shared/streams/server/plugin.ts

```diff
@@ -73,34 +79,56 @@ export class StreamsPlugin
 
     this.telemtryService.setup(core.analytics);
 
+    const alertingFeatures = STREAMS_RULE_TYPE_IDS.map((ruleTypeId) => ({
+      ruleTypeId,
+      consumers: [STREAMS_FEATURE_ID],
```

Comment:
```suggestion
      consumers: [STREAMS_CONSUMER],
```

nit: Even if they are the same value, it will be helpful to have different names because feature IDs and consumers are not related anymore. That was the case before this PR https://github.com/elastic/kibana/pull/183756, but now it is not.

Reply:
Does the producer from the rule register need to match the consumer? I'm currently using `producer: STREAMS_FEATURE_ID` (they match by value, but should it become `producer: STREAMS_CONSUMER` or even `producer: STREAMS_PRODUCER`)?

Reply:
Yes, let's match it, and yes, let's have two variables, one for the consumer and one for the producer.